src/modules/mc_att_control/mc_att_control.hpp

```diff
@@ -118,11 +121,14 @@ class MulticopterAttitudeControl : public ModuleBase<MulticopterAttitudeControl>
 
 	matrix::Vector3f _thrust_setpoint_body; /**< body frame 3D thrust vector */
 
-	float _man_yaw_sp{0.f};                 /**< current yaw setpoint in manual mode */
-	float _man_tilt_max;                    /**< maximum tilt allowed for manual flight [rad] */
+	float _hover_thrust{NAN};
+
+	float _man_yaw_sp{0.f};				/**< current yaw setpoint in manual mode */
+	float _man_tilt_max;			/**< maximum tilt allowed for manual flight [rad] */
```

Comment:
```suggestion
	float _man_tilt_max{0.f};			/**< maximum tilt allowed for manual flight [rad] */
```

I know it's unrelated, but we shouldn't have uninitialized member variables.